QUEST_LV_0100.tsv

```diff
@@ -6715,29 +6715,29 @@ QUEST_LV_0100_20150428_006714	If you are a true Revelator, you can find the divi
 QUEST_LV_0100_20150428_006715	The space movement magic field will take you to your preferred floor in the tower.{nl}I don't know if we can use the field anymore. Let's go and find out.
 QUEST_LV_0100_20150428_006716	Damn.{nl}They broke it. It is unfixable.
 QUEST_LV_0100_20150428_006717	When you activate the device which sets barriers, monsters wont't be able to come in.{nl}The supply device for the demon's power is broken, but we can input the power with other methods.
-QUEST_LV_0100_20150428_006718	We have to be careful from here.{nl}This is the place where Antares, a dangerous magician, was prisoned.{nl}We should move up to the next floor quickly.
+QUEST_LV_0100_20150428_006718	We have to be careful here.{nl}This is the place where Antares, a dangerous magician, was imprisoned.{nl}We should move up to the next floor quickly.
 QUEST_LV_0100_20150428_006719	He was a great magician before.{nl}But, his ideology was too dangerous.
-QUEST_LV_0100_20150428_006720	The reason why he was prisoned here was because he badly wanted the concentrated demons' power here.{nl}What do you think would happen to the tower when the power which was concentrated for hundreds of years explode at once?
+QUEST_LV_0100_20150428_006720	The reason why he was imprisoned here was because he strongly desired the concentrated demon's power here.{nl}What do you think would happen to the tower when the power which was concentrated for hundreds of years exploded all at once?
```

Comment:
What do you think would happen to the tower if hundreds of years of concentrated power exploded all at once?